installing/installing_azure/installing-azure-customizations.adoc

```diff
@@ -32,6 +32,13 @@ include::modules/installation-azure-config-yaml.adoc[leveloffset=+2]
 
 include::modules/installation-configure-proxy.adoc[leveloffset=+2]
 
+// Enabling Accelerated Networking during install
+include::modules/machineset-azure-enabling-accelerated-networking-new-install.adoc[leveloffset=+1]
+
+.Additional resources
+
+* See xref:../../machine_management/creating_machinesets/creating-machineset-azure.adoc#machineset-azure-accelerated-networking_creating-machineset-azure[Accelerated Networking for Microsoft Azure VMs] for more details about Accelerated Networking.
```

Comment:
Start the xref with the prepositional phrase: For more details about Accelerated Networking, see..."